lib/instrumentation/index.js

```diff
@@ -152,6 +152,14 @@ Instrumentation.prototype.bindFunction = function (original) {
   }
 }
 
+Instrumentation.prototype.bindEmitter = function (emitter) {
+  var self = this
+  var on = emitter.on
+  emitter.on = function (name, handler) {
```

Comment:
We should probably patch the `once`, `prependListener` etc alternatives as well

Reply:
Yep. I'll make sure those get added too.